Add `target` and remove `fields_under_root` from `parsers` documentation (#31387)
elastic/beats: filebeat/docs/inputs/input-kafka.asciidoc

[source,yaml]
----
- ndjson:
  target: ""
  add_error_key: true
  message_key: log
----

> *`target`*:: The name of the new field the parsed key value pairs should go. If you

Reviewer: ```suggestion
*`target`*:: he name of the new JSON object that should contain the parsed key value pairs. If you
```

> *`target`*:: he name of the new JSON object that should contain the parsed key value pairs. If you

Reviewer: ```suggestion
*`target`*:: The name of the new JSON object that should contain the parsed key value pairs. If you
```

You are a missing a `T`.
